src/mds/MDBalancer.cc

```diff
@@ -447,7 +447,7 @@ double MDBalancer::try_match(balance_state_t& state, mds_rank_t ex, double& maxe
 {
   if (maxex <= 0 || maxim <= 0) return 0.0;
 
-  double howmuch = MIN(maxex, maxim);
+  double howmuch = std::min(maxex, maxim);
```

Comment:
PR:https://github.com/ceph/ceph/pull/18124
changes std::min to MIN, std::max to MAX .. 
Why we need changing them back?

Reply:
@amitkumar50, this PR is not "changing them back"? It seems #18124 missed these?